tests/features/test_get_features.py

```diff
@@ -859,19 +859,19 @@ def test_soma_surface_areas():
 
 def test_sholl_frequency():
     assert_allclose(get_feature('sholl_frequency', NEURON),
-                    [4, 8, 8, 14, 9, 8, 7, 7])
+                    [4, 6, 10, 8, 8, 11, 7, 9, 8, 8])
```

Comment:
how were these checked to be correct?

I notice in https://github.com/BlueBrain/NeuroM/issues/663 I asked for a fake neuron to test this on, could we add one to make sure we're catching the edge cases?

Reply:
@arnaudon may be you have additional inquiries for tests here as well?

Reply:
I think we can simply check by eye on the morphologies, by plotting circles for each sholl sphere and counting? not sure it is worth adding another test morph for that.

Reply:
By engineering a morphology that has all the edge-cases, we can be more sure that we're covering them.
For instance; there was debate on what happens when a section crosses back across the Sholl spheres.